tests/components/dynalite/test_init.py

```diff
@@ -79,6 +79,71 @@ async def test_async_setup(hass):
     assert len(hass.config_entries.async_entries(dynalite.DOMAIN)) == 1
 
 
+async def test_service_request_area_preset(hass):
+    """Test requesting and area preset via service call."""
+    with patch(
+        "homeassistant.components.dynalite.bridge.DynaliteDevices.async_setup",
+        return_value=True,
+    ), patch(
+        "dynalite_devices_lib.dynalite.Dynalite.request_area_preset", return_value=True,
+    ) as mock_req_area_pres:
+        assert await async_setup_component(
+            hass,
+            dynalite.DOMAIN,
+            {
+                dynalite.DOMAIN: {
+                    dynalite.CONF_BRIDGES: [
+                        {
+                            CONF_HOST: "1.2.3.4",
+                            CONF_PORT: 1234,
+                            dynalite.CONF_AREA: {"7": {CONF_NAME: "test"}},
+                        },
+                        {CONF_HOST: "5.6.7.8", CONF_PORT: 5678},
+                    ]
+                }
+            },
+        )
+        await hass.async_block_till_done()
+        assert len(hass.config_entries.async_entries(dynalite.DOMAIN)) == 2
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)